Is there a way to see the specific error that happened? https://dev.azure.com/shine-oss/winget-pkgs/_build/results?buildId=107952&view=logs&j=68997d86-8ece-5ae8-3ac5-ccdbfa458731&t=b79910ce-dca5-5e27-ad7f-bd285eecb5a2 just says `##[error] Failed`, but I don't see how to tell what exactly failed.

--- a/manifests/m/mtkennerly/ludusavi/0.29.1/mtkennerly.ludusavi.installer.yaml
+++ b/manifests/m/mtkennerly/ludusavi/0.29.1/mtkennerly.ludusavi.installer.yaml
@@ -1,0 +1,19 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: mtkennerly.ludusavi
+PackageVersion: 0.29.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: ludusavi.exe
+  PortableCommandAlias: ludusavi
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/mtkennerly/ludusavi/releases/download/v0.29.1/ludusavi-v0.29.1-win64.zip
+  InstallerSha256: 924A72DFF6F363776F56AAD61D8F1739E6B0D6587636F1CAAFF3012B1C475D6F
+- Architecture: x86
+  InstallerUrl: https://github.com/mtkennerly/ludusavi/releases/download/v0.29.1/ludusavi-v0.29.1-win32.zip
+  InstallerSha256: 81026A67D2DEBBB50AB50952780778811655DCFCFB0F0251C1015BE5A6B2FDDD
+ManifestType: installer
+ManifestVersion: 1.9.0

--- a/manifests/m/mtkennerly/ludusavi/0.29.1/mtkennerly.ludusavi.locale.en-US.yaml
+++ b/manifests/m/mtkennerly/ludusavi/0.29.1/mtkennerly.ludusavi.locale.en-US.yaml
@@ -1,0 +1,25 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: mtkennerly.ludusavi
+PackageVersion: 0.29.1
+PackageLocale: en-US
+Publisher: mtkennerly
+PublisherUrl: https://github.com/mtkennerly
+PublisherSupportUrl: https://github.com/mtkennerly/ludusavi/issues
+Author: Matthew T. Kennerly
+PackageName: ludusavi
+PackageUrl: https://github.com/mtkennerly/ludusavi
+License: MIT
+LicenseUrl: https://github.com/mtkennerly/ludusavi/blob/master/LICENSE
+Copyright: Copyright (c) 2020 Matthew T. Kennerly (mtkennerly)
+CopyrightUrl: https://github.com/mtkennerly/ludusavi/blob/master/LICENSE
+ShortDescription: Backup tool for PC game saves
+Moniker: ludusavi
+ReleaseNotes: |-
+  * Fixed:
+    * Glob-based backup exclusions did not work correctly.
+      (This issue was introduced in v0.29.0.)
+ReleaseNotesUrl: https://github.com/mtkennerly/ludusavi/releases/tag/v0.29.1
+ManifestType: defaultLocale
+ManifestVersion: 1.9.0

--- a/manifests/m/mtkennerly/ludusavi/0.29.1/mtkennerly.ludusavi.yaml
+++ b/manifests/m/mtkennerly/ludusavi/0.29.1/mtkennerly.ludusavi.yaml
@@ -1,0 +1,8 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: mtkennerly.ludusavi
+PackageVersion: 0.29.1
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.9.0

